packages/flutter_tools/test/general.shard/macos/macos_project_migration_test.dart

```diff
@@ -218,7 +218,7 @@ keep this 2
       expect(testLogger.statusText, isEmpty);
     });
 
-    testWithoutContext('Xcode project is migrated from 10.11 to 10.14', () async {
+    testWithoutContext('Xcode project is migrated from 10.11 to 10.15', () async {
```

Comment:
curious how migration script works - say a user is on 10.11, do they run migration scripts for 10.12, 10.13, 10.14, and then finally 10.15? Or is it migrating from any versions above directly to 10.15?

Reply:
It migrates directly, but only if the deployment target is one we set in the Xcode project template. If they updated it themselves to something specific, they are on their own:

https://github.com/flutter/flutter/pull/168101/files/fd43a43386f9ce5a07aad729d7c360628a9640ac#diff-346db78c4cd00cab61b70cc48b895efee275ecbbfed6ef55e6b8ae7436375e29R38-R40